docs/Guides/Scim/Patch/SCIM-Patch-supported-operations.md

```diff
@@ -0,0 +1,124 @@
+---
+stoplight-id: fdxg63veh82v5
+---
+
+# SCIM Patch supported operations 
+
+
+Patch for email is not supported. Email is primary attribute, is unique and is immutable for a given user.
+
+**Name**: Require absolute path and operation structure as below
+
+For Replace or Add
+```json
+{
+    "op": "replace",
+    "path": 'name.familyName',
+    "value": "newFamilyName"
+    
+}
+
+```
+For remove 
+
+```json
+{
+    "op": "remove",
+    "path": 'name.familyName'    
+}
+```
+
+
+**First Level attribute of basic type :** 
+
+attribute - "nickName", "locale", "userType", "preferredLanguage", "timezone"
+
+For Replace or Add
+```json
+{
+    "op": "replace",
+    "path": attribute,
+    "value": "newFamilyName"
+    
+}
+```
+
+For remove 
+
+```json
+{
+    "op": "remove",
+    "path": attribute  
+}
+```
+
+
+**Address** : 
+Support for a single address  of type work. Valid Patch request for address.
+
+
+For Replace or Add
+```json
+{
+    "op": "replace",
+    "path": "addresses",
+    "value": {
+      "type": "work",
+      "streetAddress": "100 Universal City Plaza",
+      "locality": "Hollywood",
+      "region": "US-IA",
+      "postalCode": "91608",
+      "country": "US",
+      "formatted": "100 Universal City Plaza Hollywood, CA 91608 USA",
+      "primary": true
+    }
+    
+}
+
+{
+    "op": "replace",
+    "path": "addresses.locality",
+    "value":  "Hollywood"
+}
+```
+
+For remove 
+
+```json
+{
+    "op": "remove",
+    path: "addresses" 
+}
+
+{
+    "op": "remove",
+    "path": "addresses.locality",
+   
+}
+```
```

Comment:
While the Vendasta platform only supports a single address, I don't think that it is compatible with the SCIM standard to behave as if there will only ever be one. We should be using a list of object syntax instead.

Reply:
Test and update this in https://vendasta.jira.com/jira/software/c/projects/MRNA/issues/MRNA-422 
A/C 3. Fix the remove addresses using the path {"op": "remove", "path": "addresses[type eq \"work\"]"} instead of {"op": "remove", "path": "addresses"}

Reply:
https://vendasta.jira.com/browse/MRNA-427, this PR addressed the query